assets/export/docx/7x10/word/styles.xml

```diff
@@ -0,0 +1,1952 @@
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+   <w:style w:type="paragraph" w:customStyle="1" w:styleId="ChapterSpacer">
+        <w:name w:val="Chapter Spacer"/>
+        <w:uiPriority w:val="15"/>
+        <w:pPr>
+            <w:widowControl w:val="0"/>
+            <w:adjustRightInd w:val="0"/>
+            <w:snapToGrid w:val="0"/>
+            <w:spacing w:before="2160"/>
+            <w:jc w:val="center"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:customStyle="1" w:styleId="HeadFieldSeparator">
+        <w:name w:val="Head Field Separator"/>
+        <w:uiPriority w:val="15"/>
+        <w:pPr>
+            <w:widowControl w:val="0"/>
+            <w:adjustRightInd w:val="0"/>
+            <w:snapToGrid w:val="0"/>
+            <w:framePr w:wrap="notBeside" w:w="9000" w:h="20" w:vAnchor="text" w:hAnchor="page" w:xAlign="center" />
+            <w:spacing w:before="0"/>
+            <w:jc w:val="center"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+    <w:style w:type="paragraph" w:customStyle="1" w:styleId="HeadEnd">
+        <w:name w:val="Head End"/>
+        <w:uiPriority w:val="15"/>
+        <w:pPr>
+            <w:widowControl w:val="0"/>
+            <w:adjustRightInd w:val="0"/>
+            <w:snapToGrid w:val="0"/>
+            <w:framePr w:wrap="notBeside" w:w="9000" w:vAnchor="text" w:hAnchor="page" w:xAlign="center" />
+            <w:spacing w:before="0"/>
+            <w:jc w:val="center"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+        <w:style w:type="paragraph" w:customStyle="1" w:styleId="NodeStart">
+        <w:name w:val="Node Start"/>
+        <w:uiPriority w:val="15"/>
+        <w:pPr>
+            <w:widowControl w:val="0"/>
+            <w:adjustRightInd w:val="0"/>
+            <w:snapToGrid w:val="0"/>
+            <w:spacing w:before="0"/>
+            <w:jc w:val="center"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+        <w:style w:type="paragraph" w:customStyle="1" w:styleId="NodeEnd">
+        <w:name w:val="Node End"/>
+        <w:uiPriority w:val="15"/>
+        <w:pPr>
+            <w:widowControl w:val="0"/>
+            <w:adjustRightInd w:val="0"/>
+            <w:snapToGrid w:val="0"/>
+            <w:spacing w:before="0"/>
+            <w:jc w:val="center"/>
+        </w:pPr>
+        <w:rPr>
+            <w:sz w:val="1"/>
+            <w:szCs w:val="1"/>
+        </w:rPr>
+    </w:style>
+    <!--
+
+
+
+    Character Styles
+
+
+
+    -->
+    <!-- TODO: Replace DefaultParagraphFont references with references to a deliberate style -->
```

Comment:
Is this TODO important? Can you explain what it will do for us?

Reply:
Low-level cleanup. The more styles that can revert to the user's built-in styles, the more likely you are to have a mismatched patchwork of styles or break formatting. For example, if the user sets 'Normal' in their normal.dot to comic sans or impact or another typeface with weird spacing, and one of our styles inherits from a default style, it's going to look terrible and possibly break the layout.